docs/DeveloperGuide.md

```diff
@@ -810,12 +839,17 @@ Clear features - `Clear Recipes`, `Clear Ingredients` and `Clear Consumption` al
 Command and Parser make use of Substitutability:
 * `ClearRecipeCommand`, `ClearIngredientCommand` and `ClearConsumptionCommand` extends `Command` <br><br>
 
-The following activity diagram shows how clear operation generally works:
-![ClearActivityDiagram](images/implementation/activityDiagrams/ClearCommandActivityDiagram.png)
-
-The following sequence diagram shows how clear operation works when `execute(clearR)`, `execute(clearF)` or
+The following two diagrams shows how clear operation generally works when clear recipes: <br> 
+`execute(clearR)` <br> 
```

Comment:
i think also missed out the " " around the commands

Reply:
ok